### DISCUSSION TITLE
Browser extension / Windows Client stopped working

### DISCUSSION THREAD

**Discussion by xinput** [dani-garcia/vaultwarden#3221] [Q&A] [answered]
### Subject of the issue
Browser extension / Windows Client stopped working since a few days.  I can't tell when exactly issue started to happen but I firstly noticed on Wednesday 1st of February. Web version and mobile clients on iOS and Android still working fine. Weird thing is that Extension and Desktop version on Windows are just having an infinite loading animation when unlocking Vault.

### Deployment environment

* vaultwarden version: 1.27.0 - December 24 2022

* Install method: Docker image

* Clients used: Web vault, Firefox Browser extension, Windows Desktop Client, iOS and Android Mobile apps.
* ![Screenshot_20230203_084731](https://user-images.githubusercontent.com/11652924/216542289-794dabf0-4716-4c5d-832e-f8955143e07c.png)

* Reverse proxy and version: Nginx Proxy Manager v2.9.19 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce
open Bitwarden Desktop app and unlock Vault, alternatively open Browser extension and unlock vault

### Expected behaviour
Logins should be displayed.

### Actual behaviour
Infinite Loading animation with no further progress

**Comment by BlackDex** [comment, marked as answer]
Have you tried to logout and login again?

**Reply by xinput** [reply]
Well. I now feel completely stupid that this simple idea didn't came up to my mind.. This worked indeed. 

sorry for the inconveniences.

**Reply by BlackDex** [reply]
np :)
